apps/workspaces/actions.py

```
@@ -251,7 +252,7 @@ def post_to_integration_settings(workspace_id: int, active: bool):
         logger.error(error)
 
 
-def export_to_qbo(workspace_id, export_mode=None, expense_group_ids=[], is_direct_export:bool = False):
+def export_to_qbo(workspace_id, export_mode=None, expense_group_ids=[], is_direct_export:bool = False, triggered_by: ExpenseImportSourceEnum = None):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)